stenotype/Makefile

```diff
@@ -15,13 +15,15 @@
 DEPS=Makefile *.h
 
 DEFINES=
+SHARED_LDFLAGS=-lleveldb -lrt -laio -lpthread -lsnappy -lseccomp
+
 ifneq (,$(wildcard /usr/include/testimony.h))
 DEFINES += -DTESTIMONY
 DEPS += /usr/include/testimony.h
+SHARED_LDFLAGS +=  -ltestimony
```

Comment:
Good call on this one!  I actually noticed this separately, see https://github.com/google/stenographer/commit/00ed53d2c7744c4fa69c8ba40fa21c43e7066e6b

Also, if you pull in the recent master, the integration test should now run.